source/input_types.rst

```diff
@@ -0,0 +1,192 @@
+***********
+Input Types
+***********
+
+InVEST models accept many different types of spatial and non-spatial data. We categorize them into 12 types:
+
+.. _number:
+
+number
+------
+A scalar value. You may enter numbers in integer or floating-point format. Fractions are not allowed.
+
+Units
+~~~~~
+Most numeric inputs have units. InVEST models use standard SI units and symbols as much as possible. For definitions of these symbols, see `SI Units <https://en.wikipedia.org/wiki/International_System_of_Units#Units_and_prefixes>`_ and `units accepted for use with SI <https://en.wikipedia.org/wiki/Non-SI_units_mentioned_in_the_SI#Units_officially_accepted_for_use_with_the_SI>`_.
+
+InVEST also uses the non-SI unit *years*.
+
+When the unit is *currency*, any currency may be used, but the same currency must be used throughout the model.
+
+
+.. _ratio:
+
+ratio
+-----
+A unitless proportion in the range 0 - 1, where 0 represents "none" and 1 represents "all".
+Some ratio inputs may be less than 0 or greater than 1, while others are strictly limited to the 0-1 range.
+
+.. _percent:
+
+percent
+-------
+A unitless proportion in the range 0 - 100, where 0 represents "none" and 100 represents "all".
+Some percent inputs may be less than 0 or greater than 100, while others are strictly limited to the 0-100 range.
+
+.. _integer:
+
+integer
+-------
+A whole number (positive or negative). Integers are often used to uniquely identify something.
+Because rasters can only store numeric data, it is common to use integer codes as labels that map to non-numeric data
+(such as land use/landcover types and soil groups).
+
+.. _text:
+
+text
+----
+Freeform text. InVEST accepts any Unicode character. For best results, use Unicode character sets for non-Latin alphabets.
```

Comment:
Maybe we can say both? I feel like "unicode" is more recognizable to a non technical audience

Reply:
Personally, I'd be just fine with both.  UTF-8 is probably the best-recognized (and probably synonymous to most people), but since GDAL only takes UTF-8 (and ASCII, a subset of UTF-8), clarifying that "when we say unicode, we really mean UTF-8" could be a prudent clarification.